department/department_app/views/positions_views.py

```diff
@@ -0,0 +1,21 @@
+"""Views positions"""
+from rest_framework import generics
+from department_app.rest.serializers import PositionsListSerializer
+from department_app.models import Positions
+
+
+class PositionsCreateView(generics.CreateAPIView):
+    """Create positions"""
+    serializer_class = PositionsListSerializer
+
+
+class PositionsListView(generics.ListAPIView):
+    """Read all positions"""
+    serializer_class = PositionsListSerializer
+    queryset = Positions.objects.all()
+
+
+class PositionsDetailView(generics.RetrieveUpdateDestroyAPIView):
+    """Read, update, delete"""
+    serializer_class = PositionsListSerializer
+    queryset = Positions.objects.all()

```

Comment:
Please use ViewSet this will allow to minimize the code.

Reply:
Fixed